Find a functionally equivalent code:android.widget.TimePicker.is24HourView:COMMENT
Method Modifier: public      
Comment:/**
 * @return true if this is in 24 hour view else false.
 */

@@ -1,3 +1,3 @@
 {
-    return mIs24HourView;
+    return mDelegate.is24HourView();
 }

